deploy/examples/with-grpc-plugin.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: with-grpc-plugin
+spec:
+  storage:
+    type: grpc-plugin
+    grpcPlugin:
+      image: demo-storage-plugin:1.0.0
+      binary: "/plugin/demo-storage-plugin"
+      configurationFile: "/etc/plugin-config/config.json"
+  volumeMounts:
```

Comment:
I commented later in this PR as well, but do we need to specify the volume mounts at all? Can't it be an implementation detail in the operator?

Reply:
It was originally using a hardcoded path of `/plugin` for the volume: https://github.com/jaegertracing/jaeger-operator/commit/d02f64cb49284ae5b6b9fb26cc8b9a535860b40b#diff-a3ffdb4ba507d593652145a4af1f4bb6

https://github.com/jaegertracing/jaeger-operator/pull/1045#issuecomment-623563861

I can add it back :)

It's a choice between explicit setup by the user, or automatically set up but *still not completely hidden*, because both `binary:` in the CR and the `Dockerfile` install paths must know the name of the directory.

For me, either way is fine, both have downsides.

Reply:
I think we can go with hardcoding the /plugin for now, and see if we need to make it customizable in the future. I can't think of a reason why someone would want to customize this...